frontend/appflowy_flutter/lib/mobile/presentation/mobile_bottom_navigation_bar.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:appflowy/generated/flowy_svgs.g.dart';
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+
+/// Builds the "shell" for the app by building a Scaffold with a
+/// BottomNavigationBar, where [child] is placed in the body of the Scaffold.
+class MobileBottomNavigationBar extends StatelessWidget {
+  /// Constructs an [MobileBottomNavigationBar].
+  const MobileBottomNavigationBar({
+    required this.navigationShell,
+    Key? key,
+  }) : super(key: key ?? const ValueKey<String>('MobileBottomNavigationBar'));
```

Comment:
It's my first time seeing this code. Is it really necessary to ensure that the key is not null?

Reply:
Deleted